### PR TITLE
Adding stylelint to the admin new-theme

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | I added stylelint with bootstrap conf to the admin new-theme folder
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | It change nothing for the moment

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16758)

### CODE REVIEW

**Comment by Progi1984** [comment]
@PrestaShop/prestashop-core-developers : Shouldn't it be added to Travis ?

**Comment by NeOMakinG** [comment]
> @PrestaShop/prestashop-core-developers : Shouldn't it be added to Travis ?

We need to resolve every errors before no ? (4000)

**Comment by PierreRambaud** [comment]
@NeOMakinG Errors must be resolved before adding it to travis, same for eslint :)

**Comment by NeOMakinG** [comment]
Yes that's why I said that to @Progi1984

**Comment by Progi1984** [comment]
> > @PrestaShop/prestashop-core-developers : Shouldn't it be added to Travis ?
> 
> We need to resolve every errors before no ? (4000)

Exactly. Thank you for your contribution. :wink:

File: admin-dev/themes/new-theme/package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "test": "./node_modules/.bin/mocha  --require babel-register --reporter spec \"tests/**/*.spec.js\"",
     "build": "NODE_ENV=production webpack --progress --colors --debug --display-chunks",
     "dev": "webpack --progress --colors --debug --display-chunks --watch",
-    "watch": "npm run dev"
+    "watch": "npm run dev",
+    "scss-lint": "stylelint \"**/*.scss\" --cache --cache-location .cache/.stylelintcache --formatter verbose",
+    "scss-fix": "stylelint \"**/*.scss\" --fix --cache --cache-location .cache/.stylelintcache"
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/package.json]: Should we target the scss directory directly?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/package.json]: We can

**Pull request closed without merging**